0000-charity-welcome-doc.md

```diff
@@ -0,0 +1,34 @@
+
+If you love post mortems in general, [Dan Luu's](https://github.com/danluu/post-mortems) collection probably warms your cold little heart.
+
+Why do these outages stand out in our minds?  Possibly because the ripple effects had a big impact on your own systems, but more likely it's because you __learned something__ from the detailed, painstaking, revealing public post mortem that was published in the wake of the outage.
+
+For me, the best incident retrospectives are the one that make me cringe and realize, _"that could have been me."_  The best ones are where I come away with an actionable list of things I have learned and can use to protect my own systems.
+
+With this in mind, we are going to start publishing a series of book reports on both current and classic post-mortems.  These reviews are designed to highlight and recap some of the critical moments in our industry, so we can preserve the knowledge and collectively level up on our incident reporting skills.  Some questions we will attempt to address are:
+
+* How did affect you?  Why does it stand out for you?
+* What did you learn from it?
+* What was said (or unsaid) that made the post mortem meaningful to you?
+* Was something left unclear or dissatisfying?  What do you wish they had disclosed, or been able to disclose?
+
+A couple of book reports are already up: see [@petey5k's piece on the recent GitHub outage](https://github.com/Operations-Incident-Board/Postmortem-Report-Reviews/blob/master/2016-01-28-pshima-github-outage.md), and [@gabinante's review of the classic 2011 EBS outage](https://github.com/Operations-Incident-Board/Postmortem-Report-Reviews/blob/master/2016-03-07-gabinante-AWS-EBS-2011.md).  
+
+We appreciate the effort that everyone puts into post mortems and public retrospectives.  Writing retrospectives is hard!  Pitching the level of detail to the right audience is hard.  We hope that providing feedback on how your retrospectives were meaningful for us will give everyone more insight into how their efforts are valued and received.
+
+Yours in triumph and in tragedy,
+
+[charity.](https://github.com/charity)
+
+P.S. Want to write a book report?  Please do!!  Check out the [template](https://github.com/Operations-Incident-Board/Postmortem-Report-Reviews) and send us a pull request.  :)
```

Comment:
Should this link to a CONTRIBUTING (that doesnt exist yet)?  Just wondering if the template is clear enough for everyone to get started.